olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp

```diff
@@ -108,11 +115,11 @@ model::Partitions PartitionsCacheRepository::Get(
 boost::optional<model::Partitions> PartitionsCacheRepository::Get(
     const PartitionsRequest& request) {
   std::string hrn(hrn_.ToCatalogHRNString());
-  auto cachedIds = cache_->Get(
-      CreateKey(hrn, request.GetLayerId(), request.GetVersion()),
-      [](const std::string& serializedIds) {
-        return parser::parse<std::vector<std::string>>(serializedIds);
-      });
+  auto key = CreateKey(hrn, request.GetLayerId(), request.GetVersion());
+  LOG_TRACE_F(kLogtag, "Get '%s'", key.c_str());
```

Comment:
LOG_INFO_F?

Reply:
due to decision to put trace level at the start of a method